source/documentation/gluster-hyperconverged/chap-Maintenance_and_Upgrading_Resources.html.md

```diff
@@ -29,5 +29,28 @@ NOTE: If geo-replication is setup on the gluster volumes, the geo-replication ne
     * Replace bricks from the host to be removed using the `Replace Brick` option
     * Once all bricks are replaces, the host can be moved to maintenance and removed.
 
+## Graceful shutdown and startup
+Currently there is no scirpt or single command available to shutdown a whole cluster. However, the following steps can be executed to manually shutdown and start a cluster again.
+
+### Shutdown
+1. Shutdown all VMs
+1. Enable global ha maintenance
+1. Wait for all VMs to be down
+1. If the cluster is running a hosted engine:
```

Comment:
It probably does... how else would you hyper-converge?

Reply:
To my unstanding hyper-converge means to combine storage and computing on the same box. Since I am able to mange multiple clusters within a single engine, there is not always a need to have it self hosted.

Reply:
Interesting - so both storage and compute are on the same hosts, but the Engine is not running as part of it. OK, that's indeed a possibility, though I doubt many have it. ACK.